.github/workflows/default.yml

```
@@ -357,6 +357,7 @@ jobs:
       if: always()  # We want the log even on failures.
 
   build-sbom:
```

A release that fails the test cannot be considered a meaningful release.
Therefore, build-sbom and build-wheels also have test as a dependency.
I think sbom is similar to this, so how about having test as a dependency?

Additionally, similar to build-sbom and build-wheels, jobs responsible for creating artifacts required for release can be identified at a glance in the same layer.
```suggestion
  build-sbom:
    needs: [lint, typecheck, test]
```